libraries/net/include/graphene/net/node.hpp

```diff
@@ -199,7 +199,7 @@ namespace graphene { namespace net {
         void close();
 
         void      set_node_delegate( node_delegate* del );
-        void      set_advertise_algorithm( std::string algo, std::vector<std::string> advertise_list = std::vector<std::string>() );
+        void      set_advertise_algorithm( std::string algo, fc::optional<std::vector<std::string>> advertise_list = fc::optional<std::vector<std::string>>() );
```

Comment:
I guess const reference is better here? By the way, please wrap the long line.

Reply:
Done